src/widgets/unfold_widgets/import_widget.py

```diff
@@ -185,15 +177,17 @@ def click_listener(self, button_type: str):
 
     def load_from_file_handle(self):
         self.import_state_label.setText("Loading ...")
-        filepath = self.filepath_line.text()
-        result = self.engine.load_data_from_file(filepath)
-        if result:
-            self.import_state_label.setText(result)
-            return
-        self.clear_widgets()
-        self.set_options()
-        self.set_columns_grid()
-        self.display_data()
+        file_path: str = QFileDialog.getOpenFileName(self, 'Choose file', '.', "*.csv *.json")[0]
+        try:
+            self.engine.load_data_from_file(file_path)
+        except ValueError as e:
+            self.import_state_label.setText(str(e))
+        else:
+            self.filepath_line.setText(file_path)
+            self.clear_widgets()
+            self.set_options()
+            self.set_columns_grid()
+            self.display_data()
```

Comment:
```suggestion
        else:
            self.clear_widgets()
            self.filepath_line.setText(file_path)
            self.set_options()
            self.set_columns_grid()
            self.display_data()
```
I think we should clear the field with the file name, when we remove data from this file. So I suggest add to the `clear_data` function: `self.filepath_line.clear()` and change order in the code above.